static/app/components/roleSelectControl.tsx

```diff
@@ -17,7 +17,7 @@ type Props = Omit<ControlProps<OptionType>, 'onChange' | 'value'> & {
    * Narrower type than SelectControl because there is no empty value
    */
   onChange?: (value: OptionType) => void;
-  value?: string;
+  value?: string | null;
```

Comment:
My understanding is that `foo?: string` means that the key might not be on the object. This makes a material difference when you do `Object.keys(myObj)`. IMO on the frontend we don't do a good job on this distinction because JS shenanigans causes `myObj[foo]` with key?/undefined/null evaluates to falsey. 

The `null` value is passed from the backend when a user.